Tests/NIOSSHTests/ChildChannelMultiplexerTests.swift

```diff
@@ -886,8 +897,13 @@ final class ChildChannelMultiplexerTests: XCTestCase {
         // Issue a write to the child.
         var bytes = channel.allocator.buffer(capacity: 1024)
         bytes.writeString("Hello from the unit tests")
-        XCTAssertThrowsError(try channel.writeAndFlush(SSHChannelData(type: .channel, data: .byteBuffer(bytes))).wait())
```

Comment:
Do we need to rewrite this because of "non-sendable" warning on `SSHChannelData`? My PR should address this